mail/templates/success/body.html

```diff
@@ -0,0 +1,19 @@
+{% extends "email_base.html" %}
+{% load static %}
+
+{% block content %}
+  <tr>
+      <td style="background-color: #ffffff;">
+        <p>Hello, your video is ready!</p>
```

Comment:
```suggestion
        <p>Hello, your video is now available on the OVS web site.</p>
```
I think we can collapse the two sentences into one. They say the same thing.